docs/DeveloperGuide.adoc

```diff
@@ -98,39 +123,34 @@ image::LogicClassDiagram.png[]
 *API* :
```

Comment:
For the logic component diagram, it seems like a lot of things were changed (e.g. ArgumentMultimap, Tokenizer) but from what I've seen from the codebase, those classes seems to be still there. Is this intended?

Also, there is no interaction between the logic class and parser class, which is a little weird because typically they are related to one another.